### ISSUE TITLE
Support for customtkinter appearance mode

### ISSUE THREAD

**Issue by Alex-Schaefer** [Thisal-D/tkchart#9]
**Is your feature request related to a problem? Please describe.**
Customtkinter has the option to change between light and dark appearance mode, but this tkchart widget does not support this out of the box.
[https://customtkinter.tomschimansky.com/documentation/appearancemode](url)
Adding `customtkinter.set_appearance_mode("light")` to the 1. Simple.py example results in this:
![Screenshot 2024-03-07 151646](https://github.com/Thisal-D/tkchart/assets/81265029/58662495-e3c8-4aed-bb8f-eaaf77fe6291)

**Describe the solution you'd like**
customtkinter widgets accept a tuple with two colors, one for light and one for dark mode.
[https://customtkinter.tomschimansky.com/documentation/color](url)
If the mode gets changed while the program is running, the colors should change to match the mode.
This would be nice to have for this widget, too. Then it can be integrated in customtkinter apps with the option to change between dark and light mode.

**Comment by Thisal-D** [comment]
> **Is your feature request related to a problem? Please describe.** Customtkinter has the option to change between light and dark appearance mode, but this tkchart widget does not support this out of the box. [https://customtkinter.tomschimansky.com/documentation/appearancemode](url) Adding `customtkinter.set_appearance_mode("light")` to the 1. Simple.py example results in this: ![Screenshot 2024-03-07 151646](https://private-user-images.githubusercontent.com/81265029/310910228-58662495-e3c8-4aed-bb8f-eaaf77fe6291.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDk4MjQ1NzEsIm5iZiI6MTcwOTgyNDI3MSwicGF0aCI6Ii84MTI2NTAyOS8zMTA5MTAyMjgtNTg2NjI0OTUtZTNjOC00YWVkLWJiOGYtZWFhZjc3ZmU2MjkxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzA3VDE1MTExMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA1YzMzOWNmYjNhNjg4NzJmYmFlY2M1NzJmNWE0MTYyZTA2NzNiNDUwMTI0MzAwZjhjNmM1ODdlZWVkODA0ODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.2FHdcIgYTq6ppEDaC4ByM8VXA4eU_BQk8IA0b19sihE)
> 
> **Describe the solution you'd like** customtkinter widgets accept a tuple with two colors, one for light and one for dark mode. [https://customtkinter.tomschimansky.com/documentation/color](url) If the mode gets changed while the program is running, the colors should change to match the mode. This would be nice to have for this widget, too. Then it can be integrated in customtkinter apps with the option to change between dark and light mode.


**### First thank you for the your idea .

I appreciate your suggestion regarding tkchart. tkchart is built upon tkinter, I haven't yet found a method to directly modify its color through it. I'm still exploring a solutions.

But, if you're facing a time constraint, you can manually adjust the colors using the configure() method of individual LineChart elements.**

**Comment by Alex-Schaefer** [comment]
Yes, I think it is not the easiest thing to implement this, especially if you plan to keep the widget working for both normal tkinter and customtkinter.

Unfortunately, I don't have time for this right now, but maybe I'll create a customtkinter only fork of this next month, but can't promise anything.

**Comment by Thisal-D** [comment]
> Yes, I think it is not the easiest thing to implement this, especially if you plan to keep the widget working for both normal tkinter and customtkinter.
> 
> Unfortunately, I don't have time for this right now, but maybe I'll create a customtkinter only fork of this next month, but can't promise anything.

No worries. Maybe i can make this  using customtkinter, i'll try .

**Comment by Thisal-D** [comment]
https://github.com/Thisal-D/tkchart/assets/93121062/5e4d9650-ca2c-4b4e-8c41-ff602e72b69d

made some progress 

> Yes, I think it is not the easiest thing to implement this, especially if you plan to keep the widget working for both normal tkinter and customtkinter.
> 
> Unfortunately, I don't have time for this right now, but maybe I'll create a customtkinter only fork of this next month, but can't promise anything.

**Comment by Thisal-D** [comment]
> Yes, I think it is not the easiest thing to implement this, especially if you plan to keep the widget working for both normal tkinter and customtkinter.
> 
> Unfortunately, I don't have time for this right now, but maybe I'll create a customtkinter only fork of this next month, but can't promise anything.

There is issue with CTkCanvas, its 'bg' parameter not even support for tuple value and not support for theme changes. I think CTkCanvas is just a copy of tkinter Canvas. That's why it's not support for dark and light theme, Can you create custom CTkCanvas, support tuple value as 'bg' and track the theme changes by itself. it will be really helpfull.

**Comment by Thisal-D** [comment]
> Author

https://github.com/Thisal-D/ctkchart

try this
